activiti-engine/src/main/java/org/activiti/engine/impl/ProcessDefinitionQueryImpl.java

```diff
@@ -149,6 +150,14 @@ public ProcessDefinitionQueryImpl processDefinitionKey(String key) {
         return this;
     }
 
+    public ProcessDefinitionQueryImpl processDefinitionKeys(Set<String> keys) {
+        if (keys == null) {
```

Comment:
I'm wondering if this `if` is necessary. When you create a new instance of ProcessDefinitionQueryImpl the default value is `null` and that's a valid state. In addition, you have a check in the query itself. What do you think?

Reply:
Yeah it's not necessary, I think the idea is to reveal to the consumer code using the API that there is a mistake in the consumer code. If you've called this method and passed in null then presumably you've made a slip and intended to pass in a value as passing null would be pointless.

Reply:
Yeah, it makes sense. But in this case it would be nice to have unit tests covering that.

Reply:
Have added a test